contribute/design/1.x/maya/20190107-cStor-volume-resize.md

```diff
@@ -0,0 +1,259 @@
+---
+oep-number: draft Resize 20190701
+title: cStor Volume Resize
+authors:
+  - "@mittachaitu"
+owners:
+  - "@kmova"
+  - "@vishnuitta"
+  - "@amitkumardas"
+  - "@payes"
+  - "@pawanpraka1"
+editor: "@mittachaitu"
+creation-date: 2019-07-01
+last-updated: 2019-07-01
+status: provisional
+see-also:
+  - NA
+replaces:
+  - NA
+superseded-by:
+  - NA
+---
+
+# cStor Volume Resize
+
+## Table of Contents
+
+* [Table of Contents](#table-of-contents)
+* [Summary](#summary)
+* [Motivation](#motivation)
+    * [Goals](#goals)
+* [Proposal](#proposal)
+    * [User Stories](#user-stories)
+      * [Resize cStor Volume](#resize-cstor-volume)
+    * [Implementation Details](#implementation-details)
+    * [Custom Resources used to resize cStor volume](#custom-resource-used-to-resize)
+    * [Current Implementation](#current-implementation)
+      * [High Level cStor Volume Resize Workflow](#high-level-cStor-volume-resize-workflow)
+    * [High Level Design](#high-level-design)
+      * [CStorVolumeClaim -- existing custom resource](#cstorvolumeclaim----existing-custom-resource)
+      * [CStorVolume -- existing custom resource](#cstorvolume----existing-custom-resource)
+      * [CStorVolumeReplica -- existing custom resource](#cstorvolumereplica----existing-custom-resource)
+* [Drawbacks](#drawbacks)
+* [Alternatives](#alternatives)
+
+## Summary
+
+This proposal charts out the design details to implement resize workflow on cStor Volumes.
```

Comment:
Is this applicable for only CSI driver based cStor volumes? or does it work for both non-csi and the new ones?

Reply:
Updated informing this is only for CSI cStor volumes.